src/educational-technology.rst

```diff
@@ -1,18 +1,227 @@
-.. :Authors: - Cyrus Omar
+.. :Authors: - Cyrus Omar, Hannah Potter
 
 .. title:: Educational Technology
 
 Overview
 ========
 
-TODO: Overview
+Educational technology is focused on facilitating learning. This includes supporting novice programmers learning the fundamentals
+of programming, supplying tools for developers that promote knowledge sharing, assisting educators in their teaching objectives, and 
+facilitating learning for experienced programmers using new technologies and systems. 
+
+Programming languages and environments that are designed for use during a learning process are governed by different design criteria than those that are
+designed to be useful for experts. Educational programming environments typically focus on two problems. One is trying to flatten the 
+steep learning curve often faced by novice programmers and help them overcome what is typically perceived as "hard" with 
+programming :cite:`guzdial2004programming`. The second is supporting the motivations that different 
+people have for trying to learn these new programming skills :cite:`kelleher2005lowering`.
+
+There are a few key design principles that are often used in either of these strategies. One is to try to minimize frustration while still being
+challenging enough to maintain interest, with the related principle of incorporating motivations for learning these skills. 
+Another is related to being as concrete as possible versus abstract, as people need to understand concrete concepts before they
```

Comment:
instead of making this out to be a "versus" relationship, consider rephrasing to focus on the need to learn abstract concepts using concrete instances